session.go

```diff
@@ -66,6 +66,18 @@ var (
 	Lz4    Compression = frame.Lz4
 )
 
+func NewDefaultLogger() log.Logger {
+	return log.NewDefaultLogger()
+}
+
+func NewDebugLogger() log.Logger {
+	return log.NewDebugLogger()
+}
+
+func NewNopLogger() log.Logger {
+	return log.NopLogger{}
+}
+
```

Comment:
These shouldn't be exposed in root package, users aren't generally interested such details, default session should use default logger. Advanced users who would like to change logging details, can import log package.